flexmeasures/api/v3_0/sensors.py

```diff
@@ -103,14 +104,14 @@ def index(
         This endpoint returns all accessible sensors.
         By default, "accessible sensors" means all sensors in the same account as the current user (if they have read permission to the account).
 
-        You can also specify any other `account` (an ID parameter), if the user has read-access to that.
-        Alternatively to filtering by account, we can filter by asset tree. If you send the `asset` parameter (an ID), then all sensors on that asset or its sub-assets are looked up (if the user has read access to that asset).
+        You can also specify an `account` (an ID parameter), if the user has read access to that account. In this case, all assets under the
+        specified account will be retrieved, and the sensors associated with these assets will be returned.
 
-        In any of the two custom filtering cases above, you can
-        - not filter by both account and asset, as that is not useful - if the asset is not part of the specified account, a 422 error is raised.
-        - add the parameter flag `include_public_assets`, which adds sensors under public assets, as well.  The default value is `false`.
+        Alternatively, you can filter by asset hierarchy by providing the `asset` parameter (ID). When this is set, all sensors on the specified
+        asset and its sub-assets are retrieved, provided the user has read access to the asset.
 
-        Finally, you can use the `include_consultancy_clients` parameter to include sensors from accounts for which the current user account is a consultant. This is only possible if the user has the role of a consultant.
+        Finally, you can use the `include_consultancy_clients` parameter to include sensors from accounts for which the current user account is a consultant.
```

Comment:
We make this possible only in the case we filter for accounts, so make that clear here, too. Move this line up to below line 108 and say it works only when filtering for account.